eq/channel.h

```diff
@@ -350,11 +349,15 @@ class Channel : public fabric::Channel< Window, Channel >
      * transform the event into a config event to be send to the application
      * using Config::sendEvent().
      *
+     * @param type the pointer event type.
      * @param event the received event.
      * @return true when the event was handled, false if not.
      * @version 1.0
      */
-    EQ_API virtual bool processEvent( const Event& event );
+    EQ_API virtual bool processEvent( EventType type, SizeEvent& event );
```

Comment:
Updated doxygen to:
```
     * The task of this method is to update the channel and event as necessary,
     * and send it to the application using Config::sendEvent().
```

Aka there are use cases where the event is modified in place during processing, e.g., to set the originator ID to the channel id.